### PR TITLE
Fixing npe

### PR DESCRIPTION
**Pull request by UrsZeidler**
This npe was swallowed quite deep.

### CODE REVIEW

File: src/main/java/org/adridadou/ethereum/values/EthAddress.java [hunk 1]
```diff
@@ -45,7 +45,9 @@ public static EthAddress of(ECKey key) {
     }
 
     public static EthAddress of(final String address) {
-        if (address != null && address.startsWith("0x")) {
+        if(address==null) 
+        	return empty();
+    	if (address != null && address.startsWith("0x")) {
```
Comment (adridadou) [inline src/main/java/org/adridadou/ethereum/values/EthAddress.java:50]: no need to retest address != null, we now it isn't from the previous if

Comment (adridadou) [reply src/main/java/org/adridadou/ethereum/values/EthAddress.java:50]: also, look at codacy for the code style.

Comment (UrsZeidler) [reply src/main/java/org/adridadou/ethereum/values/EthAddress.java:50]: No this if (address != null && address.startsWith("0x")) {....
has moved down (to line 50), web3j will return null when we create a contract.

    public static EthAddress of(final String address) {
    	if(address==null)
    		return empty();

    	if (address != null && address.startsWith("0x")) {
            return of(Hex.decode(address.substring(2)));
        }
        return of(Hex.decode(address));
    }

Comment (UrsZeidler) [reply src/main/java/org/adridadou/ethereum/values/EthAddress.java:50]: The style I can fix. Of course.

**Pull request closed without merging**